workspaces/src/network/builder.rs

```diff
@@ -1,10 +1,11 @@
 use std::future::{Future, IntoFuture};
 use std::marker::PhantomData;
-use std::path::PathBuf;
 
 use crate::network::Sandbox;
 use crate::{Network, Worker};
 
+use super::server::ValidatorKeyTactic;
```

Comment:
actually, how about we just call it `ValidatorKey`. The `Tactic` part doesn't add much anyways